server/api/notes_service.go

```diff
@@ -290,6 +291,28 @@ func (s *NotesService) UploadMedia(ctx context.Context, payload *notes.UploadMed
 	}, nil
 }
 
+func (s *NotesService) DeleteMedia(ctx context.Context, payload *notes.DeleteMediaPayload) error {
```

Comment:
This appears to also need a permissions check to make sure users can only manipulate media on stations they own.